### DISCUSSION TITLE
Workflow error code 403.

### DISCUSSION THREAD

**Discussion by oGrandeK** [balta-io/forum#284] [Tira Dúvidas] [answered]
Boa noite professor, copiei seus passos no curso de git, azure e devops e quando vou publicar o workflow no step Publish aparece este erro:

> error: Response status code does not indicate success: 403 (The specified API key is invalid, has expired, or does not have permission to access the specified package.).

Segue abaixo o log completo: 

> Run dotnet nuget push "MoneyExtension/bin/Release/*.nupkg" -k *** -s https://api.nuget.org/v3/index.json
  
> Pushing MoneyExtensions.1.0.0.nupkg to 'https://www.nuget.org/api/v2/package'...
  PUT https://www.nuget.org/api/v2/package/
  Forbidden https://www.nuget.org/api/v2/package/ [6](https://github.com/oGrandeK/money-extensions/actions/runs/4410244051/jobs/7727558413#step:7:6)0[7](https://github.com/oGrandeK/money-extensions/actions/runs/4410244051/jobs/7727558413#step:7:8)ms
error: Response status code does not indicate success: 403 (The specified API key is invalid, has expired, or does not have permission to access the specified package.).

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @oGrandeK como vai?

Por favor, dá uma olhada nestes tópicos:
https://github.com/balta-io/forum/discussions/123
https://github.com/balta-io/forum/discussions/84
https://github.com/balta-io/forum/discussions/226

**Reply by oGrandeK** [reply]
Deu certo professor, muito obrigado já estava frustradão :(

**Reply by andrebaltieri** [reply]
Que isso, fica frustrado não... semana passada mesmo fiquei 3 horas em um bug simples... só não tava com cabeça pra resolver... as coisas são assim mesmo!!! 🚀

**Reply by Leandromenezes81** [reply]
Eu sigo frustrado daqui. Hehehehehe... Segui os passos do tópico #123. Entretanto, o eero segue:

![image](https://github.com/balta-io/forum/assets/76979829/33f6dfdd-25bd-4c96-b5d3-5f6114e4d77f)

Assim está meu arquivo master.yml:

![image](https://github.com/balta-io/forum/assets/76979829/d1f17aa6-427a-4dbc-861c-645581883b8a)

        
Assim está meu arquivo MoneyExtension.csproj:

![image](https://github.com/balta-io/forum/assets/76979829/a35a7e41-e11b-4ad9-b01d-808fdf49fbb4)

Please, help me!

**Reply by andrebaltieri** [reply]
@Leandromenezes81 acredito que seu problema é igual ao 226, não ao 123...
https://github.com/balta-io/forum/discussions/226

Dá uma conferida por favor...
